src/content/about/using-cache.md

```diff
@@ -3,143 +3,39 @@ title: Using the Cache
-
-TTL values:
-
-- Positive TTL values indicate in seconds how long Cloudflare should cache the asset for
-- `0` TTL will cause assets to get cached, but expire immediately (revalidate from origin every time)
-- `-1`, or any negative value will instruct Cloudflare not to cache at all
-
-#### Custom Cache Keys
-
-_This feature is available to enterprise users only._
-
-A request's cache key is what determines if two requests are "the same" for caching purposes. If a request has the same cache key as some previous request, then we can serve the same cached response for both. For more about cache keys see [Using Custom Cache Keys](https://support.cloudflare.com/hc/en-us/articles/115004290387) support article.
-
-```javascript
-// Set cache key for this request to "some-string".
-fetch(event.request, { cf: { cacheKey: 'some-string' } })
-```
-
-Normally, Cloudflare computes the cache key for a request based on the request's URL. Sometimes, though, you'd like different URLs to be treated as if they were the same for caching purposes. For example, say your web site content is hosted from both Amazon S3 and Google Cloud Storage - you have the same content in both places, and you use a Worker to randomly balance between the two. However, you don't want to end up caching two copies of your content. You could utilize custom cache keys to cache based on the original request URL rather than the subrequest URL:
-
-```javascript
-addEventListener('fetch', event => {
-  let url = new URL(event.request.url)
-  if (Math.random() < 0.5) {
-    url.hostname = 'example.s3.amazonaws.com'
-  } else {
-    url.hostname = 'example.storage.googleapis.com'
-  }
-
-  let request = new Request(url, event.request)
-  event.respondWith(
-    fetch(request, {
-      cf: { cacheKey: event.request.url },
-    }),
-  )
-})
-```
-
-Remember, Workers operating on behalf of different zones cannot affect each other's cache. You can only override cache keys when making requests within your own zone (in the above example `event.request.url` was the key stored), or requests to hosts that are not on Cloudflare. When making a request to another Cloudflare zone (e.g. belonging to a different Cloudflare customer), that zone fully controls how its own content is cached within Cloudflare; you cannot override it.
-
-### Overriding Browser Cache (Cache-Control headers)
-
-Browsers determine what to cache through the response headers sent from Cloudflare.
+### [Cache API](/reference/apis/cache)
 
-```javascript
-async function fetchAndApply(request) {
-  let response = await fetch(request)
+The Cache API creates a private cache key namespace for each zone on Cloudflare’s CDN. Cache objects are stored in the local datacenter handling the request and are not replicated to other datacenters.
```

Comment:
I'd word it something like, "The Cache API uses a cache key namespace on Cloudflare's CDN that is private to the worker's zone."

My reasoning is that the Cache API doesn't really create anything, and `caches.default`, the one interface we document, actually uses the existing `fetch()` namespace.